website/i18n/en.json

```diff
@@ -341,9 +341,6 @@
       "toastandroid": {
         "title": "ToastAndroid"
       },
-      "toolbarandroid": {
```

Comment:
Am I assuming correctly that this change was added to the commit by mistake?

Reply:
Yes, I will remove it

Reply:
So it seems that someone forget to add a change generated after building app to their PR which landed on `master`. Regardless that this change should not be a part of this PR in my opinion.

Reply:
PR update https://github.com/facebook/react-native-website/pull/1602/commits/8d87b86c9943022a26b5e9bba6bead7c173f8c6b changes restored